tools/cxbe/Xbe.cpp

```diff
@@ -1527,3 +1524,91 @@ uint08 *Xbe::GetLogoBitmap(uint32 x_dwSize)
 
     return 0;
 }
+
+// returns the count of import table entries other than xboxkrnl.exe.
```

Comment:
nit: avoid "xboxkrnl.exe" here, and instead refer just to the "kernel", so this remains valid even if `kKernelImageName` is modified

Reply:
While I agree in principle, I think for `xboxkrnl.exe` to be safely modified it'd either need to be passed as a parameter to cxbe or there'd have to be a non-backwards compatible change accompanied by a change to the import lib in the nxdk: https://github.com/XboxDev/nxdk/blob/master/lib/xboxkrnl/xboxkrnl.exe.def.

I've gone ahead and made the language agnostic, but don't personally see a need for actually making it configurable so I'm leaving it there.